proto/engine_to_bts/engine_to_bts.proto

```diff
@@ -5,19 +5,14 @@ package enginetobts;
 option go_package = "github.com/acompany-develop/QuickMPC/proto/engine_to_bts";
 
 service EngineToBts {
-    rpc GetTriples(GetTriplesRequest) returns (GetTriplesResponse) {}
+    rpc GetTriples(GetRequest) returns (GetTriplesResponse) {}
+    rpc GetRandBits(GetRequest) returns (GetRandBitsResponse) {}
 }
 
-enum Type{
-  TYPE_UNKNOWN = 0;
-  TYPE_FLOAT = 1;
-  TYPE_FIXEDPOINT = 2;
-}
-
-message GetTriplesRequest{
+message GetRequest{
+    reserved 3;
```

Comment:
No need to reserve.
The CC and BTS are currently upgraded at the same time, so that there are no conflicting numbers.
However, `reserve` is the first time I know of this, so it is informative! Thanks!

Reply:
[8596fd6](https://github.com/acompany-develop/QuickMPC/pull/254/commits/8596fd6a64e68a5f810422663a025a3af28e5800)